docs/Examples/ErrorTypes.md

```diff
@@ -21,12 +21,12 @@ Or RxSwift way:
 }
 ```
 
-You can do that by a `switch` on different `cases` of `MoyaError`. In case of an `.Underlying` error you can also get the original `NSError` and its properties, e.g. `code` to be informed about `NSURLError` types like `NSURLErrorTimedOut` or `NSURLErrorNotConnectedToInternet`
+You can do that by a `switch` on different `cases` of `MoyaError`. In case of an `.underlying` error you can also get the original `NSError` and its properties. 
+
+e.g.) code to be informed about `NSURLError` types like `NSURLErrorTimedOut` or `NSURLErrorNotConnectedToInternet`
```

Comment:
Sorry I did this quickly! I thought `code` was arbitrarily highlighted but I realize now it was talking about the error code.

I also usually put a trailing parenthesis behind my example abbreviation: `ex)` `eg)` etc. If that's a problem I'll fix it. If you're talking about adding the spacing, I felt like having the example mid sentence really flowed poorly.